src/components/modals/FormStepSaveModal.stories.js

```diff
@@ -37,45 +38,83 @@ export default {
   },
 };
 
+const render = ({
+  isOpen,
+  closeModal,
+  submissionId,
+  suspendFormUrlLifetime,
+  onSessionDestroyed,
+  onSaveConfirm,
+}) => {
+  const [_, updateArgs] = useArgs();
+  return (
+    <>
+      <OFButton appearance="primary-action-button" onClick={() => updateArgs({isOpen: true})}>
+        Open Modal
+      </OFButton>
+      <FormStepSaveModalComponent
+        isOpen={isOpen}
+        closeModal={() => {
+          closeModal();
+          updateArgs({isOpen: false});
+        }}
+        onSessionDestroyed={() => {
+          onSessionDestroyed();
+          updateArgs({isOpen: false});
+        }}
+        onSaveConfirm={() => {
+          onSaveConfirm();
+          return {ok: true};
+        }}
+        suspendFormUrl={`${BASE_URL}submissions/${submissionId}/_suspend`}
+        submissionId={submissionId}
+        suspendFormUrlLifetime={suspendFormUrlLifetime}
+        parentSelector={() => document.getElementById('storybook-root')}
+        ariaHideApp={false}
+      />
+    </>
+  );
+};
+
 export const FormStepSaveModal = {
-  render: ({
-    isOpen,
-    closeModal,
-    submissionId,
-    suspendFormUrlLifetime,
-    onSessionDestroyed,
-    onSaveConfirm,
-  }) => {
-    const [_, updateArgs] = useArgs();
-    return (
-      <>
-        <OFButton appearance="primary-action-button" onClick={() => updateArgs({isOpen: true})}>
-          Open Modal
-        </OFButton>
-        <FormStepSaveModalComponent
-          isOpen={isOpen}
-          closeModal={() => {
-            closeModal();
-            updateArgs({isOpen: false});
-          }}
-          onSessionDestroyed={() => {
-            onSessionDestroyed();
-            updateArgs({isOpen: false});
-          }}
-          onSaveConfirm={() => {
-            onSaveConfirm();
-            return {ok: true};
-          }}
-          suspendFormUrl={`${BASE_URL}submissions/${submissionId}/_suspend`}
-          submissionId={submissionId}
-          suspendFormUrlLifetime={suspendFormUrlLifetime}
-        />
-      </>
-    );
+  render,
+  args: {
+    isOpen: false,
+    submissionId: '05b5d1d6-cd5f-465b-a4b9-6f1078ffe9cc',
+    suspendFormUrlLifetime: 7,
   },
+};
+
+export const FormStepSaveModalWithErrors = {
+  render,
   args: {
     isOpen: false,
     submissionId: '05b5d1d6-cd5f-465b-a4b9-6f1078ffe9cc',
     suspendFormUrlLifetime: 7,
   },
+
+  play: async ({canvasElement, step}) => {
+    const canvas = within(canvasElement);
+
+    // alternatively, set the arg `isOpen: true`
+    const openModalButton = canvas.getByRole('button', {name: 'Open Modal'});
+    await expect(openModalButton).toBeVisible();
+    await userEvent.click(openModalButton);
+
+    const modal = await canvas.findByRole('dialog');
+    const emailInput = await within(modal).getByText('Uw e-mailadres');
+
+    await step('Test empty value', async () => {
+      await userEvent.click(emailInput);
+      await userEvent.tab();
```

Comment:
better option would be to click the submit button, that should also trigger validation and it tests that we can submit after correcting the problem (i.e. the button is clickable). That is what is originally reported as being the issue.

I also believe there is still an unfixed bug that leaves the submit button disabled if backend validation errors arrive, so I think having a separate story (with a mock returning an HTTP 400 error) + closing and re-opening the modal should assert that the submit button is in clickable state (again).